### PR TITLE
Updated the version of spanner library

### PR DESCRIPTION
**Pull request by vkedia**
Also changed the samples code to work with that.

### CODE REVIEW

File: spanner/transaction.js [hunk 1]
```diff
@@ -33,46 +33,37 @@ function readOnlyTransaction (instanceId, databaseId) {
...
+  database.runTransaction({readOnly: true}, function(err, transaction) {
 
       const queryOne = 'SELECT SingerId, AlbumId, AlbumTitle FROM Albums';
 
       // Read #1, using SQL
-      transaction.run(queryOne)
-        .then((results) => {
-          const rows = results[0];
-
+      transaction.run(queryOne, function(err, rows) {
```
Comment (ace-n) [inline spanner/transaction.js]: Avoid callbacks and the `function` keyword - stick to promises and arrow functions where possible.

File: spanner/transaction.js [hunk 2]
```diff
@@ -33,46 +33,37 @@ function readOnlyTransaction (instanceId, databaseId) {
...
+            columns: ['SingerId', 'AlbumId', 'AlbumTitle'],
+            keySet: {
+              all: true
+            }
+          };
+
+      	  // Read #2, using the `read` method. Even if changes occur
+      	  // in-between the reads, the transaction ensures that both
+          // return the same data.
+          transaction.read('Albums', queryTwo, function(err, rows) {
```
Comment (ace-n) [inline spanner/transaction.js]: Ditto callbacks v. promises

File: spanner/transaction.js [hunk 3]
```diff
@@ -96,53 +87,47 @@ function readWriteTransaction (instanceId, databaseId) {
...
-          // Gets first album's budget
-          // As above, MarketingBudget is an INT64 and comes as a string
-          const rows = results[0].map((row) => row.toJSON());
-          firstBudget = parseInt(rows[0].MarketingBudget.value);
-          console.log(`The first album's marketing budget: ${firstBudget}`);
-        })
-      ]);
-    })
-    .then(() => {
+  database.runTransaction(function(err, transaction) {
```
Comment (ace-n) [inline spanner/transaction.js]: Ditto callbacks vs. promises.

File: spanner/transaction.js [hunk 4]
```diff
@@ -159,12 +144,13 @@ function readWriteTransaction (instanceId, databaseId) {
       ]);
     })
     // Commits the transaction and send the changes to the database
-    .then(() => transaction.commit())
-    .then(() => {
-      // Logs success
-      console.log(`Successfully executed read-write transaction to transfer ${transferAmount} from Album 2 to Album 1.`);
-    });
-    // [END read_write_transaction]
+    .then(() => transaction.commit(function(err) {
```
Comment (ace-n) [inline spanner/transaction.js]: Ditto callbacks vs. promises.

File: spanner/transaction.js (hunk 3 above)
Comment (vkedia) [inline spanner/transaction.js]: database.runTransaction no longer returns a Promise. This was a change made in the latest release

File: spanner/transaction.js (hunk 4 above)
Comment (vkedia) [inline spanner/transaction.js]: I don't think commit returns a Promise.

**Comment by vkedia** [comment]
Done.

File: spanner/transaction.js [hunk 5]
```diff
@@ -96,53 +93,50 @@ function readWriteTransaction (instanceId, databaseId) {
...
-          // Gets first album's budget
-          // As above, MarketingBudget is an INT64 and comes as a string
-          const rows = results[0].map((row) => row.toJSON());
-          firstBudget = parseInt(rows[0].MarketingBudget.value);
-          console.log(`The first album's marketing budget: ${firstBudget}`);
-        })
-      ]);
-    })
-    .then(() => {
+  database.runTransaction(function (err, transaction) {
```
Comment (ace-n) [inline spanner/transaction.js]: Nit: use arrow functions.

File: spanner/transaction.js [hunk 6]
```diff
@@ -159,12 +153,15 @@ function readWriteTransaction (instanceId, databaseId) {
       ]);
     })
     // Commits the transaction and send the changes to the database
-    .then(() => transaction.commit())
-    .then(() => {
-      // Logs success
-      console.log(`Successfully executed read-write transaction to transfer ${transferAmount} from Album 2 to Album 1.`);
-    });
-    // [END read_write_transaction]
+    .then(() => transaction.commit(function (err) {
```
Comment (ace-n) [inline spanner/transaction.js]: Nit: use arrow functions

File: spanner/transaction.js (hunk 5 above)
Comment (vkedia) [inline spanner/transaction.js]: Done

File: spanner/transaction.js (hunk 6 above)
Comment (vkedia) [inline spanner/transaction.js]: Done

**Pull request merged**